cmd/install.go

```diff
@@ -155,23 +163,115 @@ func (i *installer) fileExists(filename string) bool {
 }
 
 func (i *installer) writeResolvConf() error {
-	i.log.Info("write /etc/resolv.conf")
+	const f = "/etc/resolv.conf"
+	i.log.Info("write configuration", "file", f)
 	// Must be written here because during docker build this file is synthetic
 	// FIXME enable systemd-resolved based approach again once we figured out why it does not work on the firewall
 	// most probably because the resolved must be running in the internet facing vrf.
 	// ln -sf /run/systemd/resolve/stub-resolv.conf /etc/resolv.conf
 	// in ignite this file is a symlink to /proc/net/pnp, to pass integration test, remove this first
-	err := i.fs.Remove("/etc/resolv.conf")
+	err := i.fs.Remove(f)
 	if err != nil {
-		i.log.Info("no /etc/resolv.conf present")
+		i.log.Info("config file not present", "file", f)
 	}
 
-	// FIXME migrate to dns0.eu resolvers
 	content := []byte(
 		`nameserver 8.8.8.8
 nameserver 8.8.4.4
 `)
-	return afero.WriteFile(i.fs, "/etc/resolv.conf", content, 0644)
+
+	if i.config.DNSServers != nil {
+		var s string
+		for _, dnsServer := range i.config.DNSServers {
+			s += "nameserver " + *dnsServer.IP + "\n"
+		}
+		content = []byte(s)
+
+		if i.oss != osAlmalinux {
```

Comment:
We actually overwrite the `/etc/resolved.conf` which causes systemd-resolved not to be used. So there is no need to write the `/etc/systemd/resolved.conf.d/dns.conf`.